From e72f6da3d509561b75e223667a0380772e7bd48a Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 23 Sep 2024 20:38:09 -0700
Subject: [PATCH] include locale.h for setlocale()

This issue is surfaced on systems+clang

Fixes
../gnome-console-47.0/src/main.c:29:3: error: call to undeclared function 'setlocale'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
   29 |   setlocale (LC_ALL, "");
      |   ^
../gnome-console-47.0/src/main.c:29:14: error: use of undeclared identifier 'LC_ALL'
   29 |   setlocale (LC_ALL, "");
      |              ^

Upstream-Status: Submitted [https://gitlab.gnome.org/GNOME/console/-/merge_requests/171]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index af9c5a5..5d16160 100644
--- a/src/main.c
+++ b/src/main.c
@@ -17,7 +17,7 @@
  */
 
 #include <glib/gi18n.h>
-
+#include <locale.h>
 #include <kgx.h>
 
 int
